src/index.css

```diff
@@ -3,7 +3,7 @@ body {
   font-family: 'Roboto', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  background-color: #212529;
+  background-color: #0c0d0d;
```

Comment:
Why are we doing this here and not in the theme file?

Reply:
We can do it there, too. I figured since it's the background of ever piece of the app, we'd just do it at the highest level.

Reply:
Makes sense. Just clarifying since everything else has been done in theme. The full application background-color is one-and-done so makes sense.